flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/ExecutionCheckpointingOptions.java

```diff
@@ -38,8 +38,11 @@
  * Execution {@link ConfigOption} for configuring checkpointing related parameters.
  *
  * @see CheckpointConfig
+ * @deprecated All configuration items in this class have been moved to {@link
+ *     org.apache.flink.configuration.CheckpointingOptions}.
  */
 @PublicEvolving
+@Deprecated
```

Comment:
We should also mark it as `@Documentation.ExcludeFromDocumentation` and remove `execution_checkpointing_configuration.html`

Reply:
1. After adding the annotation, it looks like the `execution checkpointing configuration.html` file needs to be deleted manually.
2. Should all descriptions related to `execution_checkpointing_configuration` in `docs/content/docs/deployment/config.md` be deleted (line 430 to 433),same as `content_hz`.

Reply:
IIUC, all configs will be showed in `checkpointing_configuration.html`, right ?
If true, I think we should remove redundant infos.

Reply:
Configurations that were added @Deprecated are not moved to `CheckpointingOptions`, and these configurations will not be displayed in `checkpointing_configuration.html`.

Reply:
@masteryhx Update has been submitted.